ztp/gitops-subscriptions/argocd/ImageRegistry.md

```diff
@@ -1,25 +1,25 @@
 Installation
 -
-1. Use SiteConfig to generate MachineConfig for disk partitioning. Make sure to modify values in the mc appropriately as it is dependent on the underlying disk.   
+1. Use SiteConfig to generate a MachineConfig for disk partitioning. Make sure to modify the values in the MC appropriately as it is dependent on the underlying disk.
```

Comment:
@pixelsoccupied, please review this file when you get the time.
Thanks!

Reply:
/lgtm